source/nanoFramework.Json.Test/Program.cs

```diff
@@ -55,7 +55,7 @@ public static void Main()
 
             DoArrayTest();
             DoSimpleObjectTest();
-            DoFloatNaNObjectTest();
+            //DoFloatNaNObjectTest();
```

Comment:
Lets keep the test, even if it fails, so that people can see...

Reply:
Agreed.

Reply:
Didn't you create the Json.Test project?

Reply:
Sorry, the nfproj seems to be totally recreated in your PR... not sure why that is...

Reply:
I think it probably has to do with the nerdbank NuGet.  I had updated to the latest beta version and didn't realized this would cause problems.  I tried to revert back to the stable version and things got really messed up.

Reply:
Figured out the issues with Json.Test - it now builds